distributions/otelcol-ebpf-profiler/README.md

```diff
@@ -0,0 +1,33 @@
+# OpenTelemetry Collector eBPF Profiling Distribution
+
+This distribution is made specifically to be used as a node agent to gather
+profiles on all processes running on the system.
+
+It contains the [eBPF profiler
+receiver](https://github.com/open-telemetry/opentelemetry-ebpf-profiler) as
+well as a subset of components from [OpenTelemetry Collector
+Core](https://github.com/open-telemetry/opentelemetry-collector) and
+[OpenTelemetry Collector
+Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+## Requirements
+
+### CGO
+
+The use of the foreign language interface to bring in dependencies requires to
+set CGO for the eBPF profiler.
+To utilize a foreign language interface for incorporating dependencies into the
+eBPF profiler, it is essential to configure and enable CGO.
+
+## Components
+
+The full list of components is available in the [manifest](manifest.yaml).
+
+### Rules for Component Inclusion
+
+- Only includes components from Contrib and Core, except for the ebpf-profiler receiver.
+- Only components that are Alpha or higher.
+- All processors, connectors, and extensions must facilitate the collection and processing of data that is generated by the eBPF profiler.
```

Comment:
It's interesting to see that there are currently no connectors that support profies.

Reply:
That's going to change soon. I have a PR adding support for the count connector.